a10_octavia/controller/worker/tasks/server_tasks.py

```diff
@@ -263,7 +250,7 @@ def execute(self, member, vthunder, pool, flavor=None):
                             return flavor
 
 
-class MemberReserveSubnetAddr(MemberBaseTask):
+class MemberReserveSubnetAddr(BaseNetworkTask):
```

Comment:
Since these are purely network tasks, it can be shifted to `a10_network_tasks` namely `ReserveSubnetAddressForMember`

Reply:
done